scripts/rabbitmq-server-ha.ocf

```diff
@@ -1537,7 +1571,45 @@ get_monitor() {
         return $rc
     fi
 
-    # Check if the rabbitmqctl control plane is alive.
+    node_health_check
+    rc=$?
+
+    return $rc
+}
+
+# Check if the rabbitmqctl control plane is alive.
+node_health_check() {
+    if [ "$OCF_RESKEY_rmq_feature_health_check" = true ]; then
+        node_health_check_local
+    else
+        node_health_check_legacy
+    fi
+}
+
+node_health_check_local() {
+    local LH="${LH} node_health_check_local():"
+    local rc
+    local rc_timeouts
+
+    local timeout=$((TIMEOUT_ARG - 2)) # give node_health_check some time to handle timeout by itself
+    su_rabbit_cmd "${OCF_RESKEY_ctl} node_health_check -t $timeout"
+    rc=$?
+
+    check_timeouts $rc "rabbit_node_health_check_timeouts" "node_health_check"
+    rc_timeouts=$?
+
+    if [ "$rc_timeouts" -eq 2 ]; then
+        master_score 0
+        return $OCF_ERR_GENERIC
+    elif [ $rc -ne 0 ]; then
```

Comment:
Here should be something like (rc_timeouts == 0 AND rc !- 0) because if (rc_timeouts == 1 AND rc == 137), that is a timeout which should be ignored